package.json

```diff
@@ -87,7 +87,8 @@
     "tap": "^15.0.1",
     "tape": "^5.0.0",
     "through2": "^4.0.0",
-    "winston": "^3.3.3"
+    "winston": "^3.3.3",
+    "transport": "./test/fixtures/transport"
```

Comment:
What is this line for?

Reply:
The transport logic supports loading modules from the scope of the parent. Instead of calling `npm link` using `spawnSync` (which did not work on Windows), I'm just adding it as a devDependency here to avoid doing anything in the test.